api/src/queries/project/project-delete-queries.ts

```diff
@@ -249,6 +249,12 @@ export const deleteProjectSpatialSQL = (projectId: number | undefined): SQLState
  * @returns {SQLStatement} sql query object
  */
 export const deleteProjectRegionSQL = (projectId: number | undefined): SQLStatement | null => {
```

Comment:
just double checking: do we really ever send this undefined? Would be better if we can make the parent check this and then this function can just specify a number, and not worry about null checks.